lib/services/data_update.dart

```diff
@@ -8,6 +8,17 @@ Future<void> updateCompletedField(id, hasCompleted) async {
       .update({'hasCompleted': hasCompleted})
       .then((value) => debugPrint("bool updated"))
       .catchError(
-        (error) => debugPrint("Failed to update user: $error"),
+        (error) => debugPrint("Failed to update bool: $error"),
+      );
+}
+
+Future<void> updateTaskContent(id, taskContent) async {
```

Comment:
just make one method, i.e update it should take a map and update it instead of making two methods doing the same thing